src/content/docs/ai-observability/intro-to-ai-monitoring.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: 'Introduction to AI Monitoring'
+metaDescription: 'AI Monitoring lets you observe the AI-layer of your tech stack, giving you a holistic overview of the health and performance of your AI-powered app.'
+freshnessValidatedDate: never
+---
+
+import apmAiMonitoringAllCapabilities from 'images/apm_screenshot-full_ai-monitoring-all-capabilities.gif'
+
+Your AI-powered app introduces new technologies into your environment that are hard to monitor, let alone understand. With AI Monitoring, you can bridge unfamiliar technologies to familiar solutions, deepening your knowledge about your AI's behavior so you can improve its performance. 
+
+<img
+    title="View your app's AI data"
+    alt="A gif that shows where to find your AI data in New Relic"
+    src={apmAiMonitoringAllCapabilities}
+/>
+
+<figcaption>
+    Go to **[one.newrelic.com](https://one.newrelic.com) > All Capabilities > AI Monitoring. Choose your entity from the AI entities page to view your data.
+</figcaption>
+
+* **Evaluate errors and bugs on the code-level.** Get total insight over how your AI processes perform and make improvements when they fail. 
+* **View feedback from your end user.** AI Monitoring correlates prompts, AI responses, and user feedback with transaction ids so you can see where your AI assistant breaks down. 
+* **Track token usage.** See when token usage spike, then optimize your AI-powered app to keep costs down. 
+
+## Get started with AI Monitoring [#start-aim]
+
+AI Monitoring exposes your AI data as a subset of APM data. To get started, you need to:
+
+* [Create a New Relic account](https://newrelic.com/signup)
```

Comment:
Do we really need to tell people to do this? I'm on the fence about it

Reply:
tbh i'm generally inclined to say no, but i figured for GA i'll have the link and see if anything comes of it. can always remove later if it's adding noise or disrupting flow

Reply:
I'll defer to you on this @akristen :)